console/src/components/Services/Settings/Actions.ts

```diff
@@ -50,41 +54,48 @@ const reloadRemoteSchemaCacheAndGetInconsistentObjectsQuery = remoteSchemaName =
   };
 };
 
-const getReloadCacheAndGetInconsistentObjectsQuery = shouldReloadRemoteSchemas => ({
+const getReloadCacheAndGetInconsistentObjectsQuery = (
+  shouldReloadRemoteSchemas: boolean
+) => ({
   type: 'bulk',
   args: [
     getReloadMetadataQuery(shouldReloadRemoteSchemas),
     inconsistentObjectsQuery,
   ],
 });
 
-export const exportMetadata = (successCb, errorCb) => (dispatch, getState) => {
+export const exportMetadata = (successCb: Function, errorCb: Function) => (
```

Comment:
So `Function` is an instance of a function, and what it does, it describes functionality that is common to all JavaScript functions.

<img width="189" alt="Screenshot 2020-06-14 at 10 52 42" src="https://user-images.githubusercontent.com/9019397/84589286-c039db80-ae2d-11ea-8e38-14b4a75bd707.png">

However, it doesn't give us any value from the TypeScript point of view, so we don't want to use it.

In this case `successCb` can be typed as `(oldMetadata?: object) => void` and `errorCb` as `(e: Error) => void`.

We also want to use `ReduxState` (from `src/types.ts`) as the first generic parameter for `ThunkDispatch`. The second generic argument can be left as `{}`. 

To type `getState` we can use `GetReduxState` defined in `src/types.ts`.

```ts
export const exportMetadata = (
  successCb: (oldMetadata: object) => void,
  errorCb: (e: Error) => void
) => (
  dispatch: ThunkDispatch<ReduxState, {}, AnyAction>,
  getState: GetReduxState
)
```

Also, please apply these changes to all the places:

`GetReduxState` instead of `() => AppState`

`ThunkDispatch<ReduxState, {}, AnyFunction>` instead of `ThunkDispatch<any, any, AnyAction>`.